x-pack/test/cloud_security_posture_functional/pages/findings.ts

```diff
@@ -24,23 +28,84 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
     };
   });
 
+  const ruleName1 = data[0].rule.name;
+  const ruleName2 = data[1].rule.name;
+
   describe('Findings Page', () => {
+    let findings: typeof pageObjects.findings;
+    let table: typeof pageObjects.findings.table;
+
     before(async () => {
-      await pageObjects.findings.index.add(findingsMock);
-      await pageObjects.findings.navigateToFindingsPage();
+      findings = pageObjects.findings;
+      table = pageObjects.findings.table;
+
+      await findings.index.add(data);
+      await findings.navigateToFindingsPage();
+      await retry.waitFor(
+        'Findings table to be loaded',
+        async () => (await table.getRowsCount()) === data.length
+      );
     });
 
     after(async () => {
-      await pageObjects.findings.index.remove();
+      await findings.index.remove();
+    });
+
+    describe('SearchBar', () => {
+      it('add filter', async () => {
+        await filterBar.addFilter('rule.name', 'is', ruleName1);
+
+        expect(await filterBar.hasFilter('rule.name', ruleName1)).to.be(true);
+        expect(await table.hasColumnValue('Rule', ruleName1)).to.be(true);
+      });
```

Comment:
changed some of our page objects to read more like `filterBar` 

i find that this:
`expect(pageObject.getSomeResult(foo)).to.be(boolean)`

reads nicer than 
`pageObject.doSomethinOrFail()` 

mosty because it tells a short story so readers can understand what's being tested better. also seems like somewhat [recommended](https://www.selenium.dev/documentation/test_practices/encouraged/page_object_models/#implementation-notes)

both are valuable though, sorting is still done like this for now